src/HoneyLocker.sol

```diff
@@ -235,27 +233,21 @@ contract HoneyLocker is TokenReceiver, Ownable {
 
     /*###############################################################*/
     function withdrawBERA(uint256 _amount) public onlyOwner {
-        address treasury = HONEY_QUEEN.treasury();
         uint256 fees = HONEY_QUEEN.computeFees(_amount);
-        STL.safeTransferETH(treasury, fees);
         STL.safeTransferETH(msg.sender, _amount - fees);
-        /*!*/
+        HONEY_QUEEN.beekeeper().distributeFees{value: fees}(referral, address(0), fees);
```

Comment:
So to confirm, fees is 2% and we dictate what percentage of that goes to an authorized referrer that we add

Reply:
The "global" fee (how much is taken out of what the user withdraws) is set on HQ, and then within BK we work with a default split but this can be customized depending on partnerships we set